crates/common/fork_choice/src/handlers.rs

```diff
@@ -235,3 +235,47 @@ pub fn on_attester_slashing(
     }
     Ok(())
 }
+
+pub fn compute_slots_since_epoch_start(slot: u64) -> u8 {
+    (slot - compute_start_slot_at_epoch(compute_epoch_at_slot(slot)))
+        .try_into()
+        .unwrap()
+}
+
+pub fn on_tick_per_slot(store: &mut Store, time: u64) -> anyhow::Result<()> {
+    let previous_slot = store.get_current_slot();
+
+    // Update store time
+    store.time = time;
+
+    let current_slot = store.get_current_slot();
+
+    // If this is a new slot, reset store.proposer_boost_root
+    if current_slot > previous_slot {
+        let _ = store.proposer_boost_root == B256::ZERO;
```

Comment:
```suggestion
        store.proposer_boost_root = B256::ZERO;
```

You are trying to assign `store.proposer_boost_root` to 0, not compare if the 2 values are equal